sql/catalyst/src/main/scala/org/apache/spark/sql/sources/filters.scala

```diff
@@ -74,6 +75,15 @@ sealed abstract class Filter {
    * Converts V1 filter to V2 filter
    */
   private[sql] def toV2: Predicate
+
+  protected def toV2Column(attribute: String): NamedReference = {
+    try {
+      FieldReference(attribute)
```

Comment:
We should add some comments to explain it.

Reply:
I will have a follow up to add some comments.